cpp/tests/transform/row_conversion.cpp

```diff
@@ -972,7 +975,7 @@ TEST_F(RowToColumnTests, ManyStrings)
     "this string is the longest string because it is duplicated more than you can imagine "
     "this string is the longest string because it is duplicated more than you can imagine "
     "this string is the longest string because it is duplicated more than you can imagine "
-    "this string is the longest string because it is duplicated more than you can imagine "
+    "this string is the longest string because it is duplicated more than you can imagine ",
     "a",
```

Comment:
I'm assuming "a" was not meant to be part of the preceding string.

Reply:
Yes, it looks like it was meant to be it's own string.